src/nrniv/netpar.cpp

```diff
@@ -152,10 +149,17 @@ void ncs_netcon_inject( int srcgid, int netconIndex, double spikeTime, bool loca
 {
     PreSyn *ps;
     NetCvode* ns = net_cvode_instance;
-    if( localNetCons )
-        gid2out_->find( srcgid, ps );
-    else
-        gid2in_->find( srcgid, ps );
+    if( localNetCons ) {
+        auto iter = gid2out_.find(srcgid);   
+        if (iter != gid2out_.end()) {
+            ps = iter->second;
+        }
+    }else{
+        auto iter = gid2in_.find(srcgid);
+        if (iter != gid2in_.end()) {
+            ps = iter->second;
+        }
+    }
```

Comment:
I think
```c++
auto const& map = localNetCons ? gid2out_ : gid2in_;
auto const iter = map.find(srcgid);
PreSyn *ps{iter != map.end ? iter->second : nullptr};
```
not important, but something like this would be a little less verbose 🤷